### PR TITLE
broadcast-api

### PR DESCRIPTION
**Pull request by chentsulin**
#186

### CODE REVIEW

File: packages/messaging-api-messenger/README.md [hunk 1]
```diff
@@ -346,7 +347,7 @@ client.sendTemplate(USER_ID, {
 
 <br />
 
-## `sendButtonTemplate(userId, title, buttons)` - [Official Docs](https://developers.facebook.com/docs/messenger-platform/send-api-reference/button-template)
+## `uttonTemplate(userId, title, buttons)` - [Official Docs](https://developers.facebook.com/docs/messenger-platform/send-api-reference/button-template)
```
Comment (tw0517tw) [inline packages/messaging-api-messenger/README.md]: uttonTemplate

File: packages/messaging-api-messenger/README.md [hunk 2]
```diff
@@ -1249,6 +1250,119 @@ client.sendBatch([
...
+- Airline boarding pass template
+- Airline check-in template
+- Airline itinerary template
+- Airline flight update template
+- Receipt template
+- Open graph template
+
+<br />
+
+## `sendBroadcastMessage(messageCreativeId, options)`
```
Comment (tw0517tw) [inline packages/messaging-api-messenger/README.md]: is 2nd argument `options` or `custom_label_id`?

File: packages/messaging-api-messenger/README.md [hunk 3]
```diff
@@ -1249,6 +1250,119 @@ client.sendBatch([
...
+
+Param             | Type     | Description
+----------------- | -------- | -----------
+messageCreativeId | `Number` | The `message_creative_id` of the message creative to send in the broadcast.
+custom_label_id   | `Number` | The id of custom label.
```
Comment (tw0517tw) [inline packages/messaging-api-messenger/README.md]: `options.custom_label_id` ?

File: packages/messaging-api-messenger/README.md [hunk 4]
```diff
@@ -1249,6 +1250,119 @@ client.sendBatch([
...
+client.sendBroadcastMessage(938461089, { custom_label_id: LABEL_ID });
+```
+
+<br />
+
+## `sendSponsoredMessage(adAcountId, args)`
+
+Param                    | Type          | Description
+------------------------ | ------------- | -----------
+args                     | `Object`      | The Object to pass into request body.
```
Comment (tw0517tw) [inline packages/messaging-api-messenger/README.md]: doc for `adAcountId`?

File: packages/messaging-api-messenger/src/MessengerClient.js [hunk 5]
```diff
@@ -68,14 +68,14 @@ function handleError(err) {
 export default class MessengerClient {
   static connect = (
     accessToken: string,
-    version?: string = 'v2.10'
+    version?: string = '2.11'
```
Comment (tw0517tw) [inline packages/messaging-api-messenger/src/MessengerClient.js]: should update document?

File: packages/messaging-api-messenger/README.md (hunk 2 above)
Comment (chentsulin) [reply packages/messaging-api-messenger/README.md]: options

**Pull request merged**